Sources/Sentry/SentryNSDataSwizzling.m

```diff
@@ -23,6 +23,11 @@ - (void)startWithOptions:(SentryOptions *)options tracker:(SentryFileIOTracker *
 {
     self.tracker = tracker;
 
+    if (!options.enableSwizzling || options.experimental.disableDataSwizzling) {
+        SENTRY_LOG_DEBUG(@"Auto-tracking of NSData is disabled")
+        return;
```

Comment:
`m`: Splitting this up into two early returns and printing one log message per option helps to understand which flag exactly was responsible for disabling the NSDataSwizzling.